### PR TITLE
Rails 5.0 compatibility

### PR DESCRIPTION
**Pull request by PolRod**
Update dependencies for rails 5, and fix deprecations.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+  - 2.2.10
+  - 2.3.7
+  - 2.4.4
+  - 2.5.3
 env:
-  - RAILS_VERSION=4.0.13
-  - RAILS_VERSION=4.1.15
-  - RAILS_VERSION=4.2.7
-before_script: gem install bundler -v 1.12.5
+  - RAILS_VERSION=5.0.7
```
Comment (camertron) [inline .travis.yml:10]: Can we also support and test against rails 4.2?

File: History.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
+== 2.0.0
```
Comment (camertron) [inline History.txt]: If all these changes are backwards-compatible, can we bump the minor version instead of the major?

**Review by camertron — CHANGES_REQUESTED** [review]
Thanks, just a few small things :)

**Comment by PolRod** [comment]
@camertron made it compatible with rails 4.2 as well, let me know what you think!

File: i18n-instrument.gemspec [hunk 3]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   if ENV['RAILS_VERSION']
     s.add_dependency 'railties', "~> #{ENV['RAILS_VERSION']}"
   else
-    s.add_dependency 'railties', '~> 4.0'
+    s.add_dependency 'railties', '<= 5.0.7', '>= 4.0'
```
Comment (camertron) [inline i18n-instrument.gemspec:21]: How about `'< 5.1.0', '>= 4.0'`?

**Review by camertron — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (andriimosin) [reply .travis.yml:10]: Support for Rails 5.1 would also be nice, so we will not update this twice

Comment (PolRod) [reply .travis.yml:10]: @andriymosin That's what I wanted too, but discovered there are some incompatibilities in this repo between 5.1 and 4.2.
My plan was to have this version as a stepping stone for moving between 4 and 5, maintaining compatibility for both.  Then we'd release another version that actually works for all 5 versions but is not compatible with 4. Does that make sense?

Comment (andriimosin) [reply .travis.yml:10]: @PolRod I see, thank you for explaining, I didn't know you faced such problems. Now it looks like the only possible solution. Thank you!

**Pull request merged**